python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -227,8 +227,8 @@ def runDataRef(self, patchRefList, subdir="", cosmos=None):
         subdir = "patch-" + str(patchList[0]) if len(patchList) == 1 else subdir
         filenamer = Filenamer(repoInfo.butler, self.outputDataset, repoInfo.dataId, subdir=subdir)
         # Find a visit/ccd input so that you can check for meas_mosaic input (i.e. to set uberCalLabel)
-        self.uberCalLabel = determineUberCalLabel(repoInfo, patchList[0], coaddName=self.config.coaddName)
-        self.log.info("Uber-calibration used: {:}".format(self.uberCalLabel))
+        self.uberCalLabel = determineExternalCalLabel(repoInfo, patchList[0], coaddName=self.config.coaddName)
+        self.log.info("External calibration(s) used: {:}".format(self.uberCalLabel))
 
```

Comment:
So there's no way of telling from the coadds what calibrations were applied, other than inferring from what exists in the repo?  I think this is a problem since we can mix and match types of calibrations.  If it's true that this sort of provenance information isn't available then we need to file a ticket.  (I just asked on slack.  I think this is important.)

Reply:
Agreed!  Sigh...